src/librustdoc/clean/mod.rs

```diff
@@ -445,11 +445,15 @@ impl Clean<GenericParamDef> for ty::GenericParamDef {
                     },
                 )
             }
-            ty::GenericParamDefKind::Const { .. } => (
+            ty::GenericParamDefKind::Const { has_default, .. } => (
                 self.name,
                 GenericParamDefKind::Const {
                     did: self.def_id,
                     ty: cx.tcx.type_of(self.def_id).clean(cx),
+                    default: match has_default {
+                        true => Some(cx.tcx.const_param_default(self.def_id).to_string()),
+                        false => None,
+                    },
```

Comment:
Sneaky boxy introducing bool matches where they are actually the best possible option... ... unless you are able to use unstable features 😜 

```suggestion
                    default: has_default.and_then(|| cx.tcx.const_param_default(self.def_id).to_string()),
```